src/main/java/uk/gov/hmcts/ccd/domain/model/search/WorkbasketInput.java

```diff
@@ -1,33 +1,5 @@
 package uk.gov.hmcts.ccd.domain.model.search;
 
-public class WorkbasketInput {
-
-    private String label;
-    private int order;
-    private Field field;
-
-    public String getLabel() {
-        return label;
-    }
-
-    public void setLabel(String label) {
-        this.label = label;
-    }
-
-    public int getOrder() {
-        return order;
-    }
-
-    public void setOrder(int order) {
-        this.order = order;
-    }
-
-    public Field getField() {
-        return field;
-    }
-
-    public void setField(Field field) {
-        this.field = field;
-    }
+public class WorkbasketInput extends CriteriaInput {
```

Comment:
are WorkbasketInput and SearchInput still needed? could we get rid of them completely?

Reply:
could be done easier with this refactoring, but this has FE effects, better handle with another task